recipes/open62541/all/conanfile.py

```diff
@@ -322,10 +322,18 @@ def _configure_cmake(self):
 
     def build(self):
         for patch in self.conan_data.get("patches", {}).get(self.version, []):
-            tools.patch(**patch)
+            tools.patch(**patch)        
```

Comment:
```suggestion
            tools.patch(**patch)
```

Reply:
@madebr I add the requirement for the ua-nodeset directly in my application "the OPC UA server" since probably I will only require it when I am creating a server using the ua-nodeset compiler.

regarding the `ua_generate_nodeset_and_datatypes` "to my knowledge" the macro was written by @Pro he has also made a nice tutorial for creating servers using nodeset files [https://opcua.rocks/step-9-using-a-custom-uanodeset-with-open62541/#comment-893](url)

Reply:
Hey @Mo-Tay 
Please see https://github.com/Mo-Tay/conan-center-index/pull/1.
I "borrowed" @Pro's example in there. It would be great if it got reduced somehow.
As a open62541 test, we're only interested in checking whether the basic functionality works.